packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -195,7 +197,10 @@ const menuStyles = (props: TProps) => (base: TBase) => {
       return base.borderColorForInput;
     })(),
     width: props.horizontalConstraint === 'auto' ? 'auto' : '100%',
-    minWidth: designTokens.constraint3,
+    minMenuWidth: props.minMenuWidth
+      ? props.minMenuWidth
+      : designTokens.constraint3,
+    maxMenuWidth: props.maxMenuWidth ?? props.maxMenuWidth,
```

Comment:
Here as well - `maxWidth`.

Reply:
All updated here: https://github.com/commercetools/ui-kit/pull/2685/commits/469dc51a1509da1549c73f2fb934cfe51103575c

Thank you.